hackathons.md

```diff
@@ -0,0 +1,563 @@
+---
+title: CockroachDB Hackathon Guide
+summary: Get Up & Running with CockroachDB at a Hackathon
+toc: false
+toc_not_nested: true
+twitter: true
+no_sidebar: true
+---
+
+Hello, hackers! Here you'll find everything you need to get up and running with CockroachDB. While this guide is lengthy, it includes not only details about deploying CockroachDB, but also helps you avoid common pitfalls when using a database.
+
+For the sake of simplicity, this guide assumes you're using:
+
+- Node.js
+- Hand-written SQL statements (instead of using an ORM)
+
+However, a lot of this information is easy to adapt if you're using different technology.
+
+{{site.data.alerts.callout_danger}}If you run into any issues, hit up the Cockroach Labs booth or check out our <a href="https://www.cockroachlabs.com/docs/stable/troubleshooting-overview.html">troubleshooting documentation</a>.{{site.data.alerts.end}}
+
+<div id="toc"></div>
+
+## Quick Start: Code Sample Repo
+
+To get you started quickly, Cockroach Labs offers [a simple Express-based Node.js app on GitHub](https://github.com/sploiselle/bankofroach).
+
+To use the app, you'll still need to install and start `cockroach`, but if you've already done that process, this will make it easy to get started.
+
+<a href="https://github.com/sploiselle/bankofroach" type="button" class="btn btn-primary" style="color:#FFF;">Get the code sample</a>
+
+## 1. Install CockroachDB
+
+To install CockroachDB, you have two options. Which you choose depends on how you want to work with your team:
+
+- **Locally** on a single machine. This is useful to get started, but can be challenging because only one person can access the database at a time.
+- **Remotely** on a cloud hosting provider. This option can give everyone on your team access to the database, but can be more difficult to configure because you have to deal with your VM's networking rules.
+
+<script>
```

Comment:
Eventually, we should move this javascript into a separate file and include it on a page via a front-matter field or something. Not important for now, though.